docs/ingestion/input-sources.md

```diff
@@ -715,6 +715,13 @@ rolled-up datasource `wikipedia_rollup` by grouping on hour, "countryName", and
  to `true` to enable a compatibility mode where the timestampSpec is ignored.
 :::
 
+The [secondary partitioning method](native-batch.md#partitionsspec) determines the requisite number of concurrent worker tasks that run in parallel to complete ingestion with the Combining input source.
+Set this value in `maxNumConcurrentSubTasks` in `tuningConfig` based on the secondary partitioning method:
+- `range` or `single_dim` partitioning: greater than or equal to 1
+- `hashed` or `dynamic` partitioning: greater than or equal to 2
+
+For more information on the `maxNumConcurrentSubTasks` field, see [Implementation considerations](native-batch.md#implementation-considerations).
```

Comment:
seems unrelated to the PR

Reply:
Yeah, not directly related to this extension, but I noticed that this text was nested inside the Iceberg section: [Iceberg Filter Object](https://druid.apache.org/docs/latest/ingestion/input-sources#iceberg-filter-object) and seemed irrelevant. So I moved it to the Druid Input source, where it seems more relevant.